tests/test_dataset.py

```diff
@@ -220,7 +220,7 @@ def test_pipeline_instruction(self):
 mocked_lane_artifact8 = NamedMock(real_name='art8', reagent_labels=['D706-D502 (GAATTCGG-ATAGAGGC)'], samples=[MockedSample(real_name='sample8', udf={})])
 mocked_idt_artifact = NamedMock(
     real_name='idt_art',
-    reagent_labels=['001A IDT-ILMN TruSeq DNA-RNA UD 96 Indexes Plate_UDI0001 (CCGCGGTT-AGCGCTAG)'],
+    reagent_labels=['001A IDT-ILMN TruSeq DNA-RNA UD 96 Indexes  Plate_UDI0001 (CCGCGGTT-AGCGCTAG)'],
```

Comment:
Unit tests still pass without this because of the `\s+` in the regex, so ideally we'd want a test for both one space and two spaces, but that would be a pain the way the tests are currently set up, so this should be fine